packages/release/activate-multi-user.service

```
@@ -0,0 +1,12 @@
+[Unit]
+Description=Isolate multi-user.target
+After=configured.target
```

As with `activate-configured`:
```suggestion
Requires=configured.target
```